### PR TITLE
Update search view to support Django 1.8

### PR DESCRIPTION
**Pull request by LilyFirefly**

### CODE REVIEW

File: autocomplete/views.py [hunk 1]
```diff
@@ -93,12 +93,12 @@ def search(request):
             or_queries = [models.Q(**{construct_search(smart_str(field_name)): smart_str(bit)})
                           for field_name in search_fields]
             other_qs = QuerySet(model)
-            other_qs.dup_select_related(qs)
+            other_qs.query.select_related = qs.query.select_related
```
Comment (LilyFirefly) [inline autocomplete/views.py:96]: See django/django#666.

**Comment by LilyFirefly** [comment]
@meshy @jturnbull Review?

**Comment by LilyFirefly** [comment]
@meshy

File: autocomplete/views.py (hunk 1 above)
Comment (meshy) [inline autocomplete/views.py:96]: `/me shivers`

**Pull request merged**